### PR TITLE
Accordion component

### PR DESCRIPTION
**Pull request by robherba**
## Summary

Adds an accordion component.

## How to review this pull request

- [x] Run `npm run develop`
- [x] Go to the accordion component page: [/story/components-accordion](http://localhost:6006/?path=/story/components-accordion--accordion)
- [x] Feel free to comment, suggest changes or improvements.

https://github.com/emulsify-ds/emulsify-web-components/assets/9342274/e4ed4f36-977c-4342-975e-c31bec360f56

### CODE REVIEW

File: src/components/accordion/accordion.stories.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Meta, StoryObj } from '@storybook/web-components';
+
+import './accordion';
+
+const meta: Meta = {
+  title: 'Components/Accordion',
+  component: 'ewc-accordion',
```
Comment (mariannuar) [inline src/components/accordion/accordion.stories.ts]: Please, change it to `emulsify-accordion`

File: src/components/accordion/accordion.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import {LitElement, html, css} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+interface AccordionItem {
+  heading: string;
+  content: string;
+  active: boolean;
+}
+
+@customElement('ewc-accordion')
```
Comment (mariannuar) [inline src/components/accordion/accordion.ts]: Please, change it to `emulsify-accordion`

File: src/components/accordion/accordion.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+    .accordion-item__heading {
+      cursor: pointer;
+      position: relative;
+      padding: var(--size-lg);
+      font-size: var(--font-size-small);
+    }
+
+    .accordion-item__heading:hover {
+      background-color: var(--color-grays-100);
```
Comment (mariannuar) [inline src/components/accordion/accordion.ts]:
```suggestion
      background-color: var(--accordion-color-header-hover);
```

File: src/components/accordion/accordion.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+  static styles = css`
+    .accordion {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: var(--size-lg);
+    }
+
+    .accordion-item {
+      border-bottom: 1px solid var(--color-grays-100);
```
Comment (mariannuar) [inline src/components/accordion/accordion.ts]:
```suggestion
      border-bottom: 1px solid var(--accordion-divider-color);
      padding-top: var(--spacing-lg);
```

**Review by mariannuar — CHANGES_REQUESTED** [review]
I think there are some things missing from the `Accordion` that is on Emulsify UI Kit. Can you please compare this component with the one on UI Kit, and check that nothing is missing, at least in styles.

**Review by mariannuar — COMMENTED** [review]
This looks and works good. There are still some things that we need to add, for example, the 'Inter' typography and use the `icon` component. Let's wait to have that component merged, to include to this one :)

**Review by cienvaras — APPROVED** [review]

**Pull request merged**